src/pipx/venv.py

```diff
@@ -162,15 +185,22 @@ def create_venv(self, venv_args: List[str], pip_args: List[str], override_shared
         override_shared -- Override installing shared libraries to the pipx shared directory (default False)
         """
         logger.info("Creating virtual environment")
-        with animate("creating virtual environment", self.do_animation):
-            cmd = [self.python, "-m", "venv"]
-            if not override_shared:
-                cmd.append("--without-pip")
+        with animate(f"creating virtual environment using {self.backend}", self.do_animation):
+            if self.backend == "venv":
+                cmd = [self.python, "-m", "venv"]
+                if not override_shared:
+                    cmd.append("--without-pip")
+            elif self.backend == "uv":
+                cmd = [path_to_exec("uv"), "venv", "--python", self.python]
+            elif self.backend == "virtualenv":
+                cmd = [path_to_exec("virtualenv"), "--python", self.python]
+                if not override_shared:
+                    cmd.append("--no-pip")
             venv_process = run_subprocess(cmd + venv_args + [str(self.root)], run_dir=str(self.root))
         subprocess_post_check(venv_process)
-
-        shared_libs.create(verbose=self.verbose, pip_args=pip_args)
-        if not override_shared:
+        if self.backend != "uv" or self.installer != "uv":
+            shared_libs.create(verbose=self.verbose, pip_args=pip_args)
+        if not override_shared and (self.backend != "uv" or self.installer != "uv"):
             pipx_pth = get_site_packages(self.python_path) / PIPX_SHARED_PTH
```

Comment:
Should the `shared` environment still be injected/exist after this or should it only be used (and thus maybe renamed) as the environment holding `pip` and allowing `"{shared_libs.python_path}" -m pip --python "{self.python_path}"` being used as an installer (rather than `"{self.python_path}" -m pip` in `_run_pip`)

Reply:
Using `--python` option sounds reasonable, but would there be any scenarios where users would prefer to keep the `.pth` file?

Reply:
The only thing that comes to mind would be if they were directly running python in an app venv to run pip instead of using `pipx runpip`.